content/en/code_analysis/faq/_index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Code Analysis Frequently Asked Questions
+description: What are the frequently asked questions about Datadog Code Analysis
+further_reading:
+- link: "/code_analysis/"
+  tag: "Documentation"
+  text: "Learn about Code Analysis"
+- link: "/code_analysis/static_analysis/"
+  tag: "Documentation"
+  text: "Learn about Static Analysis"
+- link: "/code_analysis/software_composition_analysis/"
+  tag: "Documentation"
+  text: "Learn about Software Composition Analysis"
+---
+
+## Common questions
+
+### What are Datadog-hosted scans?
+
+When using a Datadog-hosted scan, Datadog scans your code within the Datadog infrastructure. Datadog acts
```

Comment:
With Datadog-hosted scans, your code is scanned within Datadog's infrastructure as opposed to within your CI pipeline. Datadog clones your code, runs the static analyzer to perform Static Analysis and/or Software Composition Analysis, and uploads the results for you.